community/kernel/src/main/java/org/neo4j/kernel/impl/query/Neo4jTransactionalContext.java

```diff
@@ -70,6 +70,7 @@ public Neo4jTransactionalContext(
             ThreadToStatementContextBridge txBridge,
             PropertyContainerLocker locker,
             InternalTransaction initialTransaction,
+            Statement intitialStatement,
```

Comment:
What a pity this didn't work out, I liked the change.

Reply:
Yeah maybe we could resurrect it later. It lead to leaked statements because we do some stuff on a statement before creating the Neo4jTransactionalContext. Needs a little bit of work I think to restructure that.